src/components/apply/avatar-radio.tsx

```diff
@@ -0,0 +1,58 @@
+import { ChangeEvent, useState } from "react";
+import Avatar from "./avatar";
+
+type AvatarType =
+  | "Wildlife Wanderer"
+  | "City Cruiser"
+  | "Foodie Fanatic"
+  | "Beach Bum"
+  | null
+  | undefined;
+
+interface AvatarRadioProps {
+  value: AvatarType;
+  onChange: (avatar: AvatarType) => void;
+}
+
+const avatarOptions: AvatarType[] = [
+  "Wildlife Wanderer",
+  "City Cruiser",
+  "Foodie Fanatic",
+  "Beach Bum",
+];
+
+const AvatarRadioGroup = ({ value, onChange }: AvatarRadioProps) => {
+  const [selectedAvatar, setSelectedAvatar] = useState<AvatarType | null>(
+    value,
+  );
+
+  const onSelect = (e: ChangeEvent<HTMLInputElement>) => {
+    const avatar = e.target.value as AvatarType;
+    setSelectedAvatar(avatar);
+    onChange(avatar);
+  };
+
+  return (
+    <div className="grid grid-cols-2 gap-8">
+      {avatarOptions.map((option, index) => (
+        <label key={index} className="flex w-full cursor-pointer">
+          <input
+            type="radio"
+            name="avatar"
+            checked={selectedAvatar === option}
+            onChange={onSelect}
+            className="hidden"
+          />
+          <Avatar
```

Comment:
I don't know where to comment this, but the Avatar radio item needs some top and horizontal padding

actual design:
<img width="311" alt="image" src="https://github.com/user-attachments/assets/0101c8a9-6367-4b86-8d91-41678d9e3e38">

you're design:
<img width="194" alt="image" src="https://github.com/user-attachments/assets/6c6460f5-a788-4f9e-bfdf-ac5cde8e20ea">

Reply:
fixed